### PR TITLE
release 0.6.3

### PR DESCRIPTION
**Pull request by dpaez**
New release :tada:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [0.6.3] - 2020-07-14
 ### Changed
 - rename dat:// with hyper:// #169
```
Comment (juliangruber) [inline CHANGELOG.md:11]:
```suggestion
- rename dat:// to hyper:// #169
```
```suggestion
- replace dat:// with hyper:// #169
```

Comment (juliangruber) [reply CHANGELOG.md:11]: Below it says "Use module property for spec #182". What does that mean?

File: README.md [hunk 2]
```diff
@@ -224,7 +224,14 @@ Error object contains some useful properties:
 
 ## Release
 
-`npm publish`
+1. create a new branch like `{username}/release-{nextVersion}`
+2. `npm version {nextVersion}`
+    - this will prompt you with a new gh release window, you can mark it as _draft_ until PR is accepted.
+3. _acceptance_ & _merge_
```
Comment (juliangruber) [inline README.md:230]: does this mean to also create a pull request, push the branch and the tag?

Comment (dpaez) [reply README.md:230]: I'm not sure. I think npm version creates the tag, but we are publishing from `main`.

This is also a tentative process, open for improvement.

File: CHANGELOG.md (hunk 1 above)
Comment (dpaez) [reply CHANGELOG.md]: > Below it says "Use module property for spec #182". What does that mean?

We added a new module called `spec.js`, it should help tracking the different versions of the spec that we are using. The problem was that we were using the wrong property for one of the specs.

File: README.md (hunk 2 above)
Comment (juliangruber) [reply README.md:230]: Unless you use a regular merge, I think the tag will point at the wrong commit. What kind of review are you looking for here?

Comment (dpaez) [reply README.md:230]: this kind of review helps, thanks. Gonna check if the tag points to the right place after merge this. The thing is now I can't bump version and prepare the changelog for release from main (can not push from main). So if this doesn't work, I'm going to see if releasing from the ci could help.

Comment (juliangruber) [reply README.md:230]: > The thing is now I can't bump version and prepare the changelog for release from main (can not push from main)

For this reason usually push to main is allowed for admins. At least that is the case for most repos I know.

**Pull request merged**